doc/cprover-manual/assigns-clause.md

```diff
@@ -76,7 +40,7 @@ clause attached to this definition states that most of the arguments
 can be assigned. However, `myInt2`, `myPair->x`, and the first four
 elements of `myArr` should not be assigned in the body of `foo`.
 
-## Semantics
+### Semantics
```

Comment:
The array range syntax `myArr[5, lastIdx]` is not supported right?

Reply:
It's not supported anymore, remove it.